src/Core/Validators/CommandResultValidator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Bit.Core.Models.Commands;
+
+namespace Bit.Core.Validators;
+
+public static class CommandResultValidator
```

Comment:
I know you two have discussed this so I may be out of the loop, but `CommandResultValidator` is an odd name to me. You're validating the inputs of the command, not the result. `CommandRequestValidator`?

Reply:
Jimmy and I talked about this and we are POC'ing some stuff now about this.

Mostly around the difference between Commands and Validators. There are a few more POC PRs we're putting together to try to get some common types out for everyone to look at.